### DISCUSSION TITLE
QGIS Plugin?

### DISCUSSION THREAD

**Discussion by aliyilma** [pysal/momepy#573] [General]
Are there any plans for QGIS Plugin implementation? 

Thank you for providing this excellent comprehensive library. But it can be a bit complicated for newcomers like me. It took me a solid 2 weeks to utilize -at least- some of the capabilities and concepts.

Momepy has good GIS capabilities as it is very dependent on PySAL and GeoPandas. If there is a QGIS implementation, the workflow can be quite fast and flexible.

**Comment by martinfleis** [comment]
I've never considered that as my workflows are Python exclusive so I am not even sure how complicated that would be.

@anitagraser you have included movingpandas in a QGIS plugin, right? Do you have any pointers? At least so I can understand what it means.

**Reply by anitagraser** [reply]
Yes, MovingPandas is included in Trajectools v2 https://plugins.qgis.org/plugins/processing_trajectory/#plugin-versions 

You can find the code at: https://github.com/movingpandas/qgis-processing-trajectory

**Comment by martinfleis** [comment]
If this is going to happen, it will not be anytime soon. I'd first like to finish the refactoring on top of `libpysal.graph` that is currently ongoing. Then there might be some space to look at that but this is unlikely going to be a priority for any of us. If someone wants to give it a go, I am happy to collaborate but don't have a capacity to drive this at the moment. It could be a nice student project though, will keep that in mind.

**Reply by aliyilma** [reply]
Thank you very much! I wish I could make a contribution.
